.github/resources/unittests/application_default_credentials.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "your-project-id",
+  "private_key_id": "your-private-key-id",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nYour private key content\n-----END PRIVATE KEY-----\n",
+  "client_email": "your-client-email@your-project.iam.gserviceaccount.com",
+  "client_id": "your-client-id",
```

Comment:
is this file actually used by the workflow? why these dummy values?

Reply:
yeah it is, there are test failures other wise 🤷🏾  I'd like to remove this but didn't have the bandwidth to look into it more